e2e/dns/db.rpz

```diff
@@ -0,0 +1,10 @@
+$TTL 60
+@            IN    SOA  localhost. root.localhost.  (
+                          2015112501   ; serial
+                          1h           ; refresh
+                          30m          ; retry
+                          1w           ; expiry
+                          30m)         ; minimum
+                   IN     NS    localhost.
+
+httpbin.saucelabs.test.com    A        192.168.100.10
```

Comment:
I'm not a huge fan of this address, I do not like using a subdomain of test.com 

I might suggest httpbin.local?